src/main/com/fulcrologic/rad/resolvers_pathom3.cljc

```diff
@@ -0,0 +1,45 @@
+(ns com.fulcrologic.rad.resolvers-pathom3
+  (:require
+    [com.fulcrologic.rad.attributes :as attr]
+    [com.fulcrologic.rad.resolvers-common :as resolvers-common]
+    ; Pathom3 is not a dependency of fulcro-rad, and should not be
+    ; It is assumed that you have it on your classpath if you
+    ; require this ns
+    [com.wsscode.pathom3.connect.operation :as pco]
+    [taoensso.encore :as enc]
+    [taoensso.timbre :as log]))
+
+(defn just-pathom3-keys [m]
+  (into {}
+        (keep (fn [k]
+                (when (= (namespace k) "com.wsscode.pathom3.connect.operation")
+                  [k (get m k)])))
+        (keys m)))
+
+(defn attribute-resolver
+  "Generate a resolver for an attribute that specifies a :com.wsscode.pathom3.connect.operation/resolve key. Returns a resolver
+  or nil.
+
+  Accepts any keys in the com.wsscode.pathom3.connect.operation ns,
+  thus any keys defresolver accepts are valid."
+  [attr]
+  (enc/when-let [resolver        (::pco/resolve attr)
```

Comment:
What do you think about adding this key to the attributes-options ns? Like I did for the pathom 2 keys. Lets ppl know what is available, along with a bit about how to use it when they are defining attributes and using autocomplete.

Reply:
I have added `pathom3-resolve`, `pathom3-output`, `pathom3-input`, and `pathom3-transform`.

Mostly copying & pasting from what was there, and adding notes on `::auth/redact` post-processing the results and `pathom3-output` generated by default.